# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xf86-video-nouveau/hotfix-gcc14.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xf86-video-nouveau-1.0.17/src/nv_include.h.vanilla	2024-07-21 17:25:49.519777661 +0200
+++ xf86-video-nouveau-1.0.17/src/nv_include.h	2024-07-21 17:32:01.384741831 +0200
@@ -1,6 +1,8 @@
 #ifndef __NV_INCLUDE_H__
 #define __NV_INCLUDE_H__
 
+#define FB_ACCESS_WRAPPER
+
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
